### DISCUSSION TITLE
How to make pinning actions column?

### DISCUSSION THREAD

**Discussion by WinKoKoNaing** [KevinVandy/material-react-table#587] [Q&A] [answered]
https://www.material-react-table.com/docs/guides/column-pinning#column-pinning-feature-guide

I found this docs but I want to pin actions column.

**Comment by KevinVandy** [comment, marked as answer]
See the display column docs

**Reply by WinKoKoNaing** [reply]
Thank you, I got it using like that - ``columnPinning: { left: ["mrt-row-actions"] }``
